### ISSUE TITLE
Convert CRISPR_db_parser.py to command line input

### ISSUE THREAD

**Issue by mbonsma** [goyalsid/phageParser#81]
The script [`CRISPR_db_parser.py`](https://github.com/goyalsid/phageParser/blob/master/CRISPR_db_parser.py) is currently set up with the input filename hardcoded [here](https://github.com/goyalsid/phageParser/blob/master/CRISPR_db_parser.py#L16). This should be changed so that it can be run from a command prompt as

```
python CRISPR_db_parser.py filename.txt
```
or
```
cat filename.txt | CRISPR_db_parser.py 
```

The function [acc2gb.py](https://github.com/goyalsid/phageParser/blob/master/acc2gb.py) operates this way, as an example.

**Comment by mbonsma** [comment]
@sina-sed and @lisajguo are working on this!

**Comment by neaGaze** [comment]
Hey ! I am starting to work on this project, thought this issue is a very good icebreaker. I've already cloned and fixed the issue; just need to upload. Can you please brief me in uploading the modified code @mbonsma  ? Thanks !

**Comment by mbonsma** [comment]
Hi @neaGaze, thanks! 

In order to upload your changes, these are the (approximate) steps:

If you cloned goyalsid/phageParser directly, it's probably easiest to create the file directly in GitHub if you're new to Git.

1. Fork [goyalsid/phageParser](https://github.com/goyalsid/phageParser) by clicking the button on the top right that says 'Fork'. This will take you to your own copy of the repository that you can modify.
 
2. In your forked copy, click 'Create new file' in the middle-right. 
![image](https://cloud.githubusercontent.com/assets/5651370/15765281/ddc41762-2900-11e6-9b52-11ab7d186bf8.png)
Then paste your code into the box, enter a comment and click "commit". 

3. Go to the 'Pull Requests' tab and click 'Create Pull Request'. You should see the file you modified in one of the tabs. If it looks like the change you made is there, create the pull request. 

4. You're done! You should see your pull request in the [pull request tab](https://github.com/goyalsid/phageParser/pulls) of goyalsid/phageParser. 

If it doesn't work, ping me and I should be able to answer in 1-2 hours.

**Comment by mbonsma** [comment]
Fixed in PR #87. :+1:
